modelskill/comparison/_comparer_plotter.py

```diff
@@ -31,6 +31,17 @@
 from ..settings import options
 
 
+def _check_arg_length_match_nmodels(arg, n_mod):
+    if isinstance(arg, str):
+        # If arg is str, convert to list (for looping)
+        arg = [arg]
+    if arg is not None and len(arg) < n_mod:
```

Comment:
This doesn't account for the case where the user passes more arguments than n_mod. This should not be silently ignored.